### PR TITLE
Roadsdebug

### PR DESCRIPTION
**Pull request by blemia51**

### CODE REVIEW

File: tuktuktravel/src/components/container/LoginContainer.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+// }
+
+// function mapDispatchToProps(dispatch) {
+//   return {
+//     saveToken: () => dispatch({
+//       type : "CREATE_SESSION", 
+//       user: res.user,
+//       token : res.token,
+//       message : res.message
+//     })
```
Comment (rocambille) [inline tuktuktravel/src/components/container/LoginContainer.js]: dead code

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
make separate PRs for front and back please

**Comment by rocambille** [comment]
are you sure you want to delete index.js from the project?

File: tuktuktravel/src/components/container/LoginContainer.js (hunk 1 above)
Comment (blemia51) [reply tuktuktravel/src/components/container/LoginContainer.js]: done

**Comment by blemia51** [comment]
yes i will push it again

**Comment by rocambille** [comment]
euh... no

**Pull request closed without merging**